lms/djangoapps/labster_accounts/management/commands/obfuscate_user_data.py

```diff
@@ -0,0 +1,154 @@
+"""
+Management command to obfuscate users data by email or in course.
+"""
+import string
+import random
+from itertools import chain
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+
+from student.models import CourseEnrollment, CourseEnrollmentAllowed
+
+from labster_course_license.models import CourseLicense
+
+
+def action_decorator(action, field_name):
+    """
+    Decorator applies action to the provided value if value is not empty.
+    Prints logs for better process transparency.
+    """
+    def wrapper(val):
```

Comment:
please use https://docs.python.org/2/library/functools.html#functools.wraps

Reply:
Why do we need to preserve only name and docstring then? https://hynek.me/articles/decorators/
why not also args and default if any?